libs/application/templates/party-letter/src/forms/LetterApplicationForm.ts

```diff
@@ -0,0 +1,77 @@
+import {
+  buildDescriptionField,
+  buildForm,
+  buildMultiField,
+  buildRadioField,
+  buildSection,
+  buildSubmitField,
+  buildTextField,
+  Form,
+  FormModes,
+} from '@island.is/application/core'
+import { m } from './messages'
+
+const partyLetters = ['A', 'B', 'C', 'X', 'H', 'I', 'O', 'P', 'Q', 'T', 'R']
```

Comment:
Do these letters mean something in regard to the law or something? if so, I would probably move that into an enum, e.g.

```typescript
enum PartyLetters {
  PartyOfAmazing = 'A',
  PartyOfBelieving = 'B'
}
```

and so on, but not sure if that's how it works. In any case, it could be a simpler enum, I guess these letters will be important allong the application flow/process

Reply:
These are letters free to be assigned to a political party for voting.
This array is just a placeholder, we'll get this data from an external service later on